### ISSUE TITLE
Sheet

### ISSUE THREAD

**Issue by jeffsonyu** [facebookresearch/3D-Vision-and-Touch#5]

**Comment by jeffsonyu** [comment]
Hello! I noticed that the npys in scene_info folders in your dataset have the keys of "cam_pos", and you still have the sheets folder which includes the finger positions? Could you explain to me the difference between these two? Appriciate it!

**Comment by jeffsonyu** [comment]
I think I have figured out the former problem. Do the sheets npys stand for the local points touch on the finger tips? And what is the difference between sheets and the 'points' in scene_info? Do they mean different point clouds from the surfaces of the objects, and the pixels from the sensor?

**Comment by EdwardSmith1884** [comment]
Hi so there are basically 3 aspects of the training.
->  In the first I take the touch reading from a finger and the pose of the hand to learn what the local surface at the touch surface looks like. This prediction outputs a point cloud. 
-> In the second phase I convert this point cloud into a mesh "sheet" ( which is just a small mesh surface) using optimization to match the predicted point cloud. This provides a mesh "sheet" representing the surface at every touch site in the dataset.
 ->  In the third phase I use these sheets to represent the known touch information and use them to predict a full object. 

The 'points' in scene_info are the ground truth local surfaces at each touch site used to train the first phase. Because I figured people wouldn't want to train all 3 stages I provided the sheets directly in 'sheets npys' which allows you to only train or test the final stage if you want to.  

Does that make sense? Let me know if you have any other questions!
